CMakeLists.txt

```diff
@@ -1,5 +1,18 @@
 cmake_minimum_required(VERSION 3.2)
 
+# Axiom Added this 7-10-2021
+if(Vita=1)
+    message("VITASDK Defined!")
+    message("VITA BUILD!")
+    set(CMAKE_TOOLCHAIN_FILE "$ENV{VITASDK}/share/vita.toolchain.cmake" CACHE PATH "toolchain file")
+    include("$ENV{VITASDK}/share/vita.cmake" REQUIRED)
+
+    set(VITA 1)
+
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu11")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+endif()
+
```

Comment:
The `CMAKE_TOOLCHAIN_FILE` is intended to be passed externally, which TheXTech always does when you build it with given toolchain file, and this toolchain file gets passed to all sub-projects and libraries.